aedes.js

```diff
@@ -270,6 +271,10 @@ Aedes.prototype.unregisterClient = function (client) {
   this.connectedClients--
   delete this.clients[client.id]
   this.emit('clientDisconnect', client)
+  this.publish({
+    topic: '$SYS/' + this.id + '/new/clientDisconnect',
+    payload: new Buffer(client.id, 'utf8')
+  }, noop)
```

Comment:
Mosquitto doesn't provide this infomation. It provides the number of connected/disconnected clients. Mosca uses $SYS/+/new/clients &  $SYS/+/disconnect/clients  for connections & disconnections.